### DISCUSSION TITLE
openfast problem running Hydrodyn

### DISCUSSION THREAD

**Discussion by Americandream123** [OpenFAST/openfast#1881] [Q&A]
Hi, I'm using open fast to calculate the 5MW example for OC4 and I'm having the following problem when I change the platform model data to one of my own set model data: modeling
Partially submerged/spherical rods requires that the first/bottom-most element of the rod must
be completely submerged. This is not the case for MemberID 5. Thank you all for your help!
![ad1afc2123b285f089dd963635102eb](https://github.com/OpenFAST/openfast/assets/146064115/dc9c739a-99c5-4c94-84a2-f1022325b571)
![5879e3221a1f59a9d2fbe3b98ded334](https://github.com/OpenFAST/openfast/assets/146064115/dbb40ee7-5528-412f-919e-ce77f5216faf)
![7a9deac67dca5533b77603c68d28163](https://github.com/OpenFAST/openfast/assets/146064115/da23fee7-5378-418b-bf53-284063015d9b)
![ffe0af53718df313177a14f08abf633](https://github.com/OpenFAST/openfast/assets/146064115/27c40748-d194-4b0c-aab7-80accae1382e)

**Comment by Americandream123** [comment]
Hi, I'm building a 10MW model with open fast and the bottom three trusses are always cylindrical how do I build it into the rectangular shape shown below thanks for the help!
![e60065371e52ab33513b52d940c5c17](https://github.com/OpenFAST/openfast/assets/146064115/8339b093-3e5a-483d-91ce-5620736f5265)
![bf0bb96e6802f2c33df74c5c30be472](https://github.com/OpenFAST/openfast/assets/146064115/416e1dbe-8612-4d13-88ff-7c883da416ca)
![6df43f514504c2ade49d8f377bc8083](https://github.com/OpenFAST/openfast/assets/146064115/807e150d-7818-43c1-9ecc-bc7f2ad4e80e)

**Reply by jjonkman** [reply]
It appears that you asked the same question in the following OpenFAST issue: https://github.com/OpenFAST/openfast/issues/1875.  Your questions have been answered there.
